airflow/plugins_manager.py

```diff
@@ -122,3 +135,15 @@ def make_module(name, objects):
     admin_views.extend(p.admin_views)
     flask_blueprints.extend(p.flask_blueprints)
     menu_links.extend(p.menu_links)
+
+# Merge base extra links with operators' own extra links
+for operators_module in operators_modules:
+    for operator in operators_module._objects:
+        operator_extra_links = []
+        operator_extra_links.extend(BaseOperator.extra_links)
+        operator_extra_links.extend(operator.extra_links)
+        operator.extra_links = list(set(operator_extra_links))
```

Comment:
We can remove this code. We only need the `functions` now

Reply:
Fixed.